Python/Product/PyDebugAttach/PyDebugAttach.cpp

```diff
@@ -1098,7 +1073,19 @@ bool DoAttach(HMODULE module, ConnectionInfo& connInfo, bool isDebug) {
         auto debugId = PyObjectHolder(isDebug, strFromString(connInfo.Buffer->DebugId));
         auto debugOptions = PyObjectHolder(isDebug, strFromString(connInfo.Buffer->DebugOptions));
         DecRef(call(attach_process.ToPython(), pyPortNum.ToPython(), debugId.ToPython(), debugOptions.ToPython(), pyTrue, pyFalse, NULL), isDebug);
+        if (auto err = errOccurred()) {
+            PyObject *type, *value, *traceback;
+            pyErrFetch(&type, &value, &traceback);
+
+            auto repr = PyObjectHolder(isDebug, pyObjectRepr(value));
+            wchar_t reprText[0x1000] = {};
+            pyUnicodeAsWideChar(repr.ToPython(), reprText, _countof(reprText) - 1);
+            fputws(reprText, stderr);
```

Comment:
Future plans are to propagate the entire text of this error to the caller - this will require switching to something like JSON to communicate attach results, instead of the current system with simple error codes.